modules/services/repositories/src/main/java/au/com/shiftyjelly/pocketcasts/repositories/sync/SyncManagerImpl.kt

```diff
@@ -419,9 +419,8 @@ class SyncManagerImpl @Inject constructor(
     private suspend fun downloadTokens(
         email: String,
         refreshToken: RefreshToken,
-        syncServerManager: SyncServerManager,
```

Comment:
This didn't need to be passed because the class already has the `SyncServerManager` stored as a field.